src/init.ts

```diff
@@ -64,7 +63,7 @@ async function checkIfConfigExists() {
         platforms.push(val);
       }
     });
-    const answer = await askYesNo(
+    const answer: boolean = await Confirm.prompt(
```

Comment:
interesting. I guess this works on Linux as well? But looks good to me.

Reply:
Yes, we used it already. That's why I decide to use it.
This confirm prompt is also used when asking to install some azure extension (I think you wrote that)